diff --git a/lib/util/mayContainChildComponent.js b/lib/util/mayContainChildComponent.js
--- a/lib/util/mayContainChildComponent.js
+++ b/lib/util/mayContainChildComponent.js
@@ -8,1 +8,1 @@
-var _minimatch = _interopRequireDefault(require("minimatch"));
+var _minimatch = _interopRequireDefault(require("minimatch").minimatch);
diff --git a/lib/util/mayHaveAccessibleLabel.js b/lib/util/mayHaveAccessibleLabel.js
--- a/lib/util/mayHaveAccessibleLabel.js
+++ b/lib/util/mayHaveAccessibleLabel.js
@@ -9,1 +9,1 @@
-var _minimatch = _interopRequireDefault(require("minimatch"));
+var _minimatch = _interopRequireDefault(require("minimatch").minimatch);
